docs/Feeds.md

```diff
@@ -182,15 +182,16 @@ Parser: Bitcash Blocklist Feed (`intelmq.bots.parsers.bitcash.parser`)
 Blocklist provided by bitcash.cz of banned IPs for service abuse, this includes scanning, sniffing, harvesting, and dos attacks.
 
 
-## BitSight Ciberfeed Stream
+## BitSight Cyberfeed Stream
```

Comment:
+1

It's officially called: "Cyberfeed" with a "y".

https://www.anubisnetworks.com/products/threat-intelligence/cyberfeed